tests/testthat/test-recall_data.R

```diff
@@ -0,0 +1,206 @@
+test_that("recall_by_make returns the correct shape",{
+    result <- recall_by_make('Nissan')
+    expect_type(result, "list")
+    expect_equal(nrow(result), 25)
+    expected_col_names <- c("Recall number","Manufacturer Name",
+                            "Model name", "Make name", "Year","Recall date")
+    expect_equal(colnames(result), expected_col_names)
+    expected_dtypes <- c("character", "character", "character",
+                         "character", "integer",   "Date")
+    returned_dtypes <- as.vector(unlist(sapply(result, class)))
```

Comment:
do you need `unlist()` here?

Reply:
applies to all following `unlist()`

Reply:
do i need it? i dont actually know lol
it was a solution to `sapply(result, class)` giving me a list with named entries, not allowing me to test the equality

Reply:
removed them, and it works as intended